hw/top_earlgrey/data/ip/chip_usbdev_testplan.hjson

```diff
@@ -271,7 +305,145 @@
       tests: []
       bazel: []
     }
-    // This test list does not yet cover Suspend, Resume, Sleep and Wake
-    // functionality.
+    {
+      name: chip_sw_usbdev_suspend_resume
+      desc: '''Verify that the USB device can detect Suspend and Resume Signaling
+            from the host.
+
+            - Host configures the USB device as normal.
+            - Host requests that the USB device enter a Suspended state, by not sending
```

Comment:
nit: It enters suspended state when there is no activity at all (i.e. always idle / J).

`host_lost` was meant to trigger when usbdev sees activity but isn't successfully receiving SOF packets for clock calibration. It's not supposed to be part of the ordinary suspend / resume flow.

Reply:
Sure. Reworded.

In an effort to keep the SiVal tests simpler and specific to IP blocks rather than them being chip-wide integration tests like the usbdev suspend/sleep/wake/resume tests previously constructed, the aim is to exercise the aon_wake module without involving the pwrmgr, sleep, power cycling etc.

Perversely the aon_wake module cannot actually report the detection of non-Idle state except via the wakeup signal to pwrmgr (software must infer that it has been awoken and regained control because of Resume Signaling, from the fact that neither 'Disconnect' nor 'Bus Reset' is indicated!), so I've dropped 'chip_sw_usbdev_aon_wake_resume' for now.


Aside: since Suspend is driven by, as you say, Idle state for > 3ms rather than absence of SOF for that time, I find myself wondering whether the presence of a Low Speed device on the same hub prevents/should prevent a device from being suspended? Propagation of Low Speed traffic (remember the fun with BIt Stuffing errors?) seems 'less than ideal' in this instance. It doesn't cause any problems, I suppose, since as far as the host is concerned the device _is_ suspended, and so the Resume Signaling must be performed anyway.